src/SFA.DAS.ApprenticeFeedback.Jobs/Functions/ApprenticeFeedbackSummaryTimerTrigger.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.Azure.WebJobs;
+using Microsoft.Extensions.Logging;
+using NServiceBus;
+using SFA.DAS.ApprenticeFeedback.Jobs.Domain.Messages.Commands;
+using SFA.DAS.ApprenticeFeedback.Jobs.Infrastructure;
+using System;
+
+namespace SFA.DAS.ApprenticeFeedback.Jobs.Functions
+{
+    public class ApprenticeFeedbackSummaryTimerTrigger
+    {
+        private readonly IFunctionEndpoint endpoint;
+
+        public ApprenticeFeedbackSummaryTimerTrigger(IFunctionEndpoint endpoint) => this.endpoint = endpoint;
+
+        [FunctionName("GenerateApprenticeFeedbackSummaries")]
+        public void ApprenticeFeedbackSummary([TimerTrigger("0 */3 * * * *")] TimerInfo timer, ExecutionContext executionContext, ILogger logger)
```

Comment:
Should the cron config in the attribute be configurable instead of hard coded ?

Reply:
Discussed offline, added Http trigger for manual intervention with set as once every 3 hours on the function for calculation